radixdlt-core/radixdlt/src/main/java/com/radixdlt/client/AccountAddress.java

```diff
@@ -0,0 +1,82 @@
+/*
+ * (C) Copyright 2021 Radix DLT Ltd
+ *
+ * Radix DLT Ltd licenses this file to you under the Apache License,
+ * Version 2.0 (the "License"); you may not use this file except in
+ * compliance with the License.  You may obtain a copy of the
+ * License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND,
+ * either express or implied.  See the License for the specific
+ * language governing permissions and limitations under the License.
+ *
+ */
+
+package com.radixdlt.client;
+
+import com.radixdlt.identifiers.REAddr;
+import com.radixdlt.serialization.DeserializeException;
+import com.radixdlt.utils.Bits;
+import com.radixdlt.utils.functional.Result;
+import org.bitcoinj.core.AddressFormatException;
+import org.bitcoinj.core.Bech32;
+
+/**
+ * Bech-32 encoding/decoding of account addresses.
+ *
+ * The human-readable part is "rdx" for mainnet, "brx" for betanet.
+ *
+ * The data part is a conversion of the 1-34 byte Radix Engine address
```

Comment:
the unit test vector in `AccountAddressTest.java`:
```java
"03" + "03".repeat(26), "brx1qvpsxqcrqvpsxqcrqvpsxqcrqvpsxqcrqvpsxqcrqvpsytf8zx",
```

contradicts the statement here : `1-34 byte Radix Engine address`, because that test is only 26 bytes, right? I wold like to see 33 bytes enforced.

Reply:
Wording is confusing. 1-34 bytes refers to the RadixEngineAddress which is a low level address, compared to the bech32 account address which maps to a RadixEngineAddress.